### ISSUE TITLE
[Bug]Force enable zoom doesn't properly work when trying to zoom on Instagram album content

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#6840]
## Steps to reproduce
Navigate to any Instagram profile
Open an album
Try to swipe zoom in from the picture area from that album 

### Expected behavior
Should zoom in/out properly from the picture area

### Actual behavior
Can't zoom in from the picture area, only from the description or comments section.

### Device information

* Android device: Google Pixel 3a (Android 9)
* Fenix version:  Nightly Build #13310607 from 11/27

### Notes
Previously mentioned in: #2477 

✔️ Works properly on: One picture posts, videos and IG TV videos
![20191127_182804](https://user-images.githubusercontent.com/51314259/69741806-5b928a00-1144-11ea-903c-0672f4edd02b.gif)

❌ Doesn't work on albums
![20191127_182843](https://user-images.githubusercontent.com/51314259/69741924-9268a000-1144-11ea-8058-ab2abe5c9fac.gif)

**Comment by AndiAJ** [comment]
Reproducible on 3.0.0-beta.2 Build #13390410 from 12/5 using a Google Pixel 3a (Android 9)

**Comment by vesta0** [comment]
This happens when a site has javascript touch gestures. ENG decisions required in terms of behaviour, will be postponed for a later date.

**Comment by sblatz** [comment]
I believe this is likely a GV issue?
